plugins/woocommerce-admin/client/core-profiler/style.scss

```diff
@@ -122,3 +124,70 @@
 		}
 	}
 }
+
+.woocommerce-profiler-business-location {
+	.woocommerce-profiler-heading {
+		margin-top: 72px;
+	}
+	.woocommerce-profiler-go-to-mystore__button {
+		width: 400px;
+		display: flex;
+		justify-content: center;
+	}
+	.components-base-control__field {
+		height: 40px;
+	}
+	.woocommerce-profiler-select-control__country {
+		width: 400px;
+		margin-bottom: 32px;
+		.woocommerce-select-control__control {
+			height: 40px;
+		}
+		#woocommerce-select-control__listbox-0 {
+			top: 40px !important;
+		}
+	}
+	.woocommerce-select-control__control-icon {
+		display: none;
+	}
+
+	.woocommerce-select-control__control.is-active {
+		.components-base-control__label {
+			display: none;
+		}
+	}
+
+	.woocommerce-select-control.is-searchable
+		.woocommerce-select-control__control-input {
+		margin: 0;
+		padding: 0;
+	}
+
+	.woocommerce-select-control.is-searchable .components-base-control__label {
+		left: 13px;
+	}
+}
+
+.woocommerce-profiler-loader {
```

Comment:
Make it responsive?

![Screenshot 2023-05-09 at 16 15 37](https://user-images.githubusercontent.com/4344253/237036536-058f1de6-bb81-4632-8eb1-44aeaa7bfa94.png)